docs/general/ddl/data-types.rst

```diff
@@ -2041,6 +2041,21 @@ to days and hours::
         SELECT 1 row in set (... sec)
 
 
+.. NOTE::
+
+    Comparisons between intervals except for equality (``=``) and
+    inequality(``<>``) are not allowed.
+
+.. NOTE::
+
+    Intervals cannot be used in :ref:`ORDER BY <sql-select-order-by>` clause.
+
+.. NOTE::
+
+    Intervals cannot be used in :ref:`MIN <aggregation-min>` and
+    :ref:`MAX <aggregation-max>` aggregations.
+
```

Comment:
Minor: Since all notes are related to INTERVAL, could merge them into a single note.